.release-please-manifest.json

```diff
@@ -1,3 +1,3 @@
 {
-    ".": "0.11.0"
+    ".": "0.12.0"
 }
```

Comment:
An empty line is missing at the end here. It should not have any impact here but it would be great to include so git does not show `\ No newline at end of file`, or a big `-` in GH web interface.